src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs

```diff
@@ -120,7 +120,7 @@ class ClassifiedResult
                 [ClassificationTypeNames.VerbatimStringLiteral] = SemanticHighlightClassification.VerbatimStringLiteral,
                 [ClassificationTypeNames.StringEscapeCharacter] = SemanticHighlightClassification.StringEscapeCharacter,
                 [ClassificationTypeNames.ClassName] = SemanticHighlightClassification.ClassName,
-                [ClassificationTypeNames.RecordName] = SemanticHighlightClassification.ClassName,
+                [ClassificationTypeNames.RecordClassName] = SemanticHighlightClassification.ClassName,
```

Comment:
Do we need to keep `RecordName` for backwards compatability?

Reply:
This maps from the Roslyn classification to the semanitic classifications we send to the client. The ClassificationTypeName comes from Roslyn which no longer contains a RecordName type. We haven't introduced a RecordName SemanticHighlightClassification yet partly because we knew that Roslyn would be introducing RecordClassName and a RecordStructName in the future.

Reply:
cool, I just wanted to make sure!